e2e/framework/utils/s3_operations.go

```diff
@@ -22,19 +22,26 @@ type S3Helper struct {
 	s3client *s3.S3
 }
 
+// create a s3 client for specfied endpoint and creds
 func CreateNewS3Helper(endpoint string, keyId string, keySecret string) *S3Helper {
 
 	creds := credentials.NewStaticCredentials(keyId, keySecret, "")
 
+	// create aws s3 config, must use 'other-v2-signature' region for ceph object store
 	awsConfig := aws.NewConfig().
 		WithRegion("other-v2-signature").
 		WithCredentials(creds).
 		WithEndpoint(endpoint).
 		WithS3ForcePathStyle(true).
 		WithDisableSSL(true)
 
+	//create new session
 	ses := session.New()
+
+	//create new s3 client connection
 	c := s3.New(ses, awsConfig)
+
+	// user a custom signer - need to convert newer amz s3 requests to v2 requests since
```

Comment:
is this comment meant to be continued? it looks like its cut off...

I had made a comment earlier on this PR but i don't see it anymore, so I'm not sure where it went...but is this complicated/additional logic in the `sign` function due to a limitation in RGW or in our s3 library?  Is there any way to avoid having to write and maintain all this logic?